deploy/service.yaml

```diff
@@ -0,0 +1,186 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Docs service template for thethingsindustries.com
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterLabels:
+      CPU:
+        default: CPU limit of a container
+      DockerImage:
+        default: Docker image
+      Memory:
+        default: Memory limit of a container
+      TaskDesiredCount:
+        default: Number of container instances to run (prod only)
+Parameters:
+  CPU:
+    AllowedPattern: .+
+    Default: '256'
+    Type: String
+  DockerImage:
+    AllowedPattern: .+
+    Default: thethingsindustries/thethingsindustries:docs
+    Type: String
+  Memory:
+    AllowedPattern: .+
```

Comment:
This can't just take anything. It needs to be a number, and considering we're on FARGATE, there are only a couple of possibilities.

Reply:
Valid ranges depend on CPU https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html

Reply:
Sure, but values such as `AUTO` or `42.42` are invalid for sure

Reply:
And will result in an immediate error, not a silent failure

Reply:
Still, it's better if CloudFormation catches it before even trying to update the stacks